odl/test/solvers/nonsmooth/difference_convex_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2014-2018 The ODL contributors
+#
+# This file is part of ODL
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tests for the d.c. solvers."""
+
+from __future__ import division
+import odl
+from odl.solvers import (dca, prox_dca, doubleprox_dc)
+import numpy as np
+import pytest
+
+
+# Places for the accepted error when comparing results
+HIGH_ACCURACY = 8
+LOW_ACCURACY = 4
+
+
+def test_dca():
+    """Test the dca method for the following simple problem:
+
+    Let a > 0, and let b be a real number. Solve
+
+    min_x a/2 (x - b)^2 - |x|
+
+    For finding possible (local) solutions, we consider several cases:
+    * x > 0 ==> ∂|.|(x) = 1, i.e., a necessary optimality condition is
```

Comment:
I guess we need a unicode marker for this

Reply:
Done.